charts/airflow/values.yaml

```diff
@@ -903,6 +921,7 @@ workers:
   ##
   autoscaling:
     enabled: false
+    apiVersion: autoscaling/v2beta2
```

Comment:
I think we should set the default as `autoscaling/v2` because it was [added in 1.23](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#horizontalpodautoscaler-v125), which is already the oldest in-support version of Kubernetes.